test/integration-spec.yaml

```
@@ -804,6 +804,30 @@
     httpsRequests:
       - "https://httpbin.org/basic-auth/foo/bar"
 
+# protocols
+- url: "https://127.0.0.1:8889"
+  label: "localhost but https version"
+  options:
+    ignore-ssl-errors: true
```

```suggestion
    ignoreSslErrors: true
```

`commander` module "camelCases" CLI options.